### DISCUSSION TITLE
Import Key files from file system (SD, internal memory)

### DISCUSSION THREAD

**Discussion by username-mendoza** [ikarus23/MifareClassicTool#442] [Ideas]
Can the feature to import key files from file system/SD/Internal memory be added? Because now i can only edit or create new key file. So, now, if i have my own key file, i can only create new (or edit existing) key file, and add keys manually, which is kind of acceptable if i have few keys, but if i have a lot of them, that starts to irritate...

**Comment by username-mendoza** [comment]
Stupid me, there is already this feature :) it's just not in same place as key files... 😅

**Reply by ikarus23** [reply]
Yeah, sorry for the bad UI. I will try to add an import/export button to the key file/dump editor and to the key file/dump file chooser.

**Comment by ikarus23** [comment]
If somebody else is searching: The import/export feature is under tools (for now).

**Comment by JoraKey12** [comment]
If you're rooted, you can browse the data folder of the app, and there should be the dictionary files...
You can add,edit or remove dictionary files, as long as you are rooted...

Just my 2 cents...

**Reply by ikarus23** [reply]
Yes, all key files and dumps are just separated into different folders. You can edit them as you like, because there are just simple text files.
